config/product/logging.js

```diff
@@ -36,4 +38,7 @@ export function init(store) {
     route:      { name: 'c-cluster-logging' },
     exact:       true,
   });
+
+  headers(LOGGING.OUTPUT, [STATE, NAME_COL, LOGGING_OUTPUT_PROVIDERS, AGE]);
```

Comment:
These are missing the namespace column (after name)

Reply:
Do we actually want that? It's grouped by namespace.

Reply:
Grouping by namespace should be optional (like it is on every other type).  Which works by removing the `NAMESPACE` column from the list if grouping is enabled.

Reply:
I understand that it's optional. I'm trying to understand where this request is coming from.

I think my confusion is stemming from the fact that the NAMESPACE column isn't actually working on any of the types that it's been added to.

I navigated to Ingress, secret, workload etc and each was grouped by namespace and didn't have a namespace column. I just looked at explorer.js and noticed that a lot of these types are actually supposed to have this column.

I'll look into fixing that.

Reply:
Nevermind, I see that only specific cases will show the column.